Dedicated Cloud Compute/getting started/create-server.md

```diff
@@ -0,0 +1,109 @@
+{{{
+  "title": "Create Server",
+  "date": "04-19-2017",
+  "author": "Anthony Hakim",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Description
+
+The User Interface (UI) for Dedicated Cloud Compute (DCC) has been updated with a new look and feel. This documents the process to create a new server.
+
+### Audience
+
+Dedicated Cloud Compute customers.
+
+### Assumptions
+
+* You have a SavvisStation Portal account and can logon successfully.
+
+### Add Server Process
+
+1. Logon and go to the new DCC UI. For more information, please refer to the [Logon to New UI](//www.ctl.io/Dedicate Cloud Compute/getting started/logon-new-UI) KB article.
```

Comment:
Is this supposed to link to a getting started guide? If so use this link:
`https://www.ctl.io/guides/dedicated-cloud-compute/logon-new-ui/`